test/framework/universal_cluster.go

```diff
@@ -119,6 +119,12 @@ func (c *UniversalCluster) DeployKuma(mode core.CpMode, opt ...KumaDeploymentOpt
 
 	env := map[string]string{"KUMA_MODE": mode, "KUMA_DNS_SERVER_PORT": "53"}
 
+	if Config.IPV6 {
+		env["KUMA_DNS_SERVER_CIDR"] = "fd00:fd00::/64"
+		env["KUMA_IPAM_MESH_SERVICE_CIDR"] = "fd00:fd01::/64"
+		env["KUMA_IPAM_MESH_EXTERNAL_SERVICE_CIDR"] = "fd00:fd02::/64"
```

Comment:
do we need to change anything outside of `test/` for this to work on a "real cluster"? like here https://github.com/kumahq/kuma/blob/c26bfce8c7597e0fad0564bb274c78d7da404f9d/pkg/config/app/kuma-cp/config.go#L288 ?

Reply:
Yes, we need add docs to set CIDR to IPV6 just like described here for KUMA_DNS_SERVER_CIDR https://kuma.io/docs/2.7.x/production/dp-config/ipv6/#ipv6-support
Here is the ticket https://github.com/kumahq/kuma-website/issues/1817